service-api/module/Application/src/Model/Entity/IdMethodIncludingNation.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Application\Model\Entity;
+
+use Exception;
+use Laminas\Form\Annotation;
+use Laminas\Form\Annotation\Validator;
+use Laminas\Validator\NotEmpty;
+
+/**
+ * DTO for holding ID method data
+ * @psalm-suppress MissingConstructor
+ * Needed here due to false positive from Laminas’s uninitialised properties
+ * @psalm-suppress UnusedProperty
+ */
+class IdMethodIncludingNation extends Entity
+{
+    #[Annotation\Required(false)]
+    #[Annotation\Validator(NotEmpty::class, options: [NotEmpty::NULL])]
```

Comment:
Could use the `Enum` validator here to check that it's a valid enum value:
```suggestion
    #[Annotation\Validator(Enum::class, options: ['enum' => IdMethod::class])]
```